frontend/src/components/ProductDescription.js

```diff
@@ -0,0 +1,15 @@
+const ProductDescription = ({ product }) => {
+  const productDetails = product.description.specs;
+
+  return (
+    <>
+      <p>{product.description.main}</p>
+      {productDetails.map(prodDetail => (
+        <p>- {prodDetail}</p>
+      ))}
```

Comment:
I think you want an unordered list here. That way you don't have to manually add the `-`
```suggestion
      <ul>
      {productDetails.map(prodDetail => (
        <li>{prodDetail}</li>
      ))}
      </ul>
```

Reply:
Ah yea, good point. Will do that. Thanks.